tests/foreman/ui/test_classparameters.py

```diff
@@ -0,0 +1,705 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Smart/Puppet Class Parameter"""
+
+from robottelo.decorators import run_only_on, stubbed, tier3, tier1
+from robottelo.test import UITestCase
+
+
+class ClassParameters(UITestCase):
+    """Implements Smart Class Parameter tests in UI"""
+
+    @stubbed()
+    @run_only_on('sat')
+    @tier3
+    def test_positive_override_checkbox(self):
+        """@test: Override the Default Parameter value.
+
+        @feature: Class Parameters - Override
+
+        @steps:
```

Comment:
Inorder for the doc to display this correctly, please leave an empty line after this line.

Example:
```py
"""
 @steps:

    1. Check the override checkobx
"""

Please update all instances
```